plugins.json

```diff
@@ -119,15 +119,27 @@
         "remote_install": true,
         "compatible_version": ">=1.0.42"
     },
+    "MarkdownPlus": {
+        "title": "MarkdownPlus",
+        "version": "1.0.0",
+        "author": "Craig Crosby",
+        "license": "MIT",
+        "description": "Improved Markdown, with check boxes, emoji shortcode, inline html, etc...",
+        "homepage": "https://github.com/creecros/MarkdownPlus",
+        "readme": "https://raw.githubusercontent.com/creecros/MarkdownPlus/master/README.md,
```

Comment:
There is missing double quote here (invalid JSON).

Reply:
and all set